tcp/coder/coder_test.go

```diff
@@ -106,3 +106,13 @@ func BenchmarkUnmarshalMessage(b *testing.B) {
 		}
 	}
 }
+
+func FuzzDecode(f *testing.F) {
+	f.Add([]byte{211, 0, 1, 1, 2, 3, 177, 97, 1, 98, 1, 99, 1, 100, 1, 101, 16, 255, 1})
+	f.Add([]byte{211, 0, 1, 1, 2, 3, 177, 97, 1, 98, 1, 99, 1, 100, 1, 101, 16, 255, 1})
```

Comment:
Testing the same decode twice doesn't seem necessary, any reason to keep the second line?

Reply:
Thanks for pointing this out.